application/btsieve/src/bin/btsieve.rs

```diff
@@ -52,18 +52,21 @@ fn main() -> Result<(), failure::Error> {
 
     let bitcoin_routes = create_bitcoin_routes(&mut runtime, settings.bitcoin)?;
 
-    let (ethereum_routes, _event_loop_handle) =
-        create_ethereum_routes(&mut runtime, settings.ethereum)?;
+    let (ethereum_routes, _event_loop) = create_ethereum_routes(&mut runtime, settings.ethereum)?;
 
-    let routes = bitcoin_routes.or(ethereum_routes);
+    let log = warp::log("btsieve::api");
+    let ping_200 = warp::path("ping").map(warp::reply);
+    let ping_route = warp::get2().and(ping_200);
```

Comment:
❓ What's this for?

Reply:
It's an extra endpoint to test if btsieve is up and running.

Reply:
What's it for in regards to completing this issue?

🔧 If nothing I think remove it.

Reply:
I'm running a test where neither bitcoin nor ethereum is connected and I've added this route in order to be able check that btsieve started up correctly.

Reply:
🔧 I am going to be the devil(@thomaseizinger)'s advocate here and suggest you replace this endpoint with `/health`. Looking at the RFC https://tools.ietf.org/id/draft-inadarei-api-health-check-01.html the response body only requires a `status` field so I think it'd be worth just implementing that and leaving the rest (`details`) to when someone needs it (as part of a future API improvement).

Reply:
As discussed offline: this endpoint currently only serves as a small helper for our tests and we implement the linked  [RFC](https://tools.ietf.org/id/draft-inadarei-api-health-check-01.html). While it sounds indeed interesting, we should discuss this feature a bit more in detail, e.g. I can imagine returning information about the currently connected blockchains, database (if we have one), etc.